### PR TITLE
Setup codacy rules and fix issues

### PR DESCRIPTION
**Pull request by NullableInt**
Using remarkjs to lint Markdown

### CODE REVIEW

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 6
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108&bid=18291964)

File: index.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ This is a normal paragraph following a header. GitHub is a code hosting platform
 
 ### Header 3
 
+### Header 3
```
Comment (payex-dev) [inline index.md:29]: Codacy Issue found: [[no-duplicate-headings-in-section] Do not use headings with similar content per section (27:1)](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108)

Comment (payex-dev) [inline index.md:29]: Codacy Issue found: [[no-duplicate-headings] Do not use headings with similar content (27:1)](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108&bid=18291964)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108&bid=18291964)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108&bid=18291964)

File: index.md [hunk 2]
```diff
@@ -208,37 +211,37 @@ and try out PayEx Checkout for yourself!' %}
...
+*   Item zip
 
 ## And an ordered list
 
 1. Item one
-1. Item two
-1. Item three
-1. Item four
+2. Item two
+3. Item three
```
Comment (payex-dev) [inline index.md:227]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 32
- Added 8
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108&bid=18291964)

File: index.md [hunk 3]
```diff
@@ -208,37 +211,37 @@ and try out PayEx Checkout for yourself!' %}
...
-  * level 2 item
-  * level 2 item
-* level 1 item
-  * level 2 item
-  * level 2 item
-* level 1 item
+*   level 1 item
+    *   level 2 item
+    *   level 2 item
+      *   level 3 item
```
Comment (payex-dev) [inline index.md:235]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108)

File: index.md [hunk 4]
```diff
@@ -208,37 +211,37 @@ and try out PayEx Checkout for yourself!' %}
...
-* level 1 item
-  * level 2 item
-  * level 2 item
-  * level 2 item
-* level 1 item
-  * level 2 item
-  * level 2 item
-* level 1 item
+*   level 1 item
+    *   level 2 item
```
Comment (payex-dev) [inline index.md:233]: Codacy Issue found: [[no-missing-blank-lines] Missing blank line before block node](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108)

File: index.md [hunk 5]
```diff
@@ -175,22 +180,20 @@ sequenceDiagram
...
-                      icon='check_circle'
-                      body='This is a successful alert.' %}
+{% include alert.html type='success' icon='check_circle'
+body='This is a successful alert.' %}
 
-{% include alert.html icon='info_outline'
-                      header='**Informational** *alert*'
-                      body='This is an **informational** alert *with* `<markdown/>`{:.language-html}.' %}
+{% include alert.html icon='info_outline' header='**Informational**     *alert*'
+body='This is an **informational**   alert *with*   `<markdown/>`{:.language-html}.'
```
Comment (payex-dev) [inline index.md:187]: Codacy Issue found: [[emphasis-marker] Emphasis should use `_` as a marker](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108)

File: index.md [hunk 6]
```diff
@@ -208,37 +211,37 @@ and try out PayEx Checkout for yourself!' %}
...
 
 ## And an ordered list
 
 1. Item one
-1. Item two
-1. Item three
-1. Item four
+2. Item two
+3. Item three
+4. Item four
```
Comment (payex-dev) [inline index.md:228]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108)

File: index.md [hunk 7]
```diff
@@ -208,37 +211,37 @@ and try out PayEx Checkout for yourself!' %}
...
+*   Item baz
+*   Item zip
 
 ## And an ordered list
 
 1. Item one
-1. Item two
-1. Item three
-1. Item four
+2. Item two
```
Comment (payex-dev) [inline index.md:226]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 1 space](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108)

File: index.md [hunk 8]
```diff
@@ -175,22 +180,20 @@ sequenceDiagram
...
-{% include alert.html type='success'
-                      icon='check_circle'
-                      body='This is a successful alert.' %}
+{% include alert.html type='success' icon='check_circle'
+body='This is a successful alert.' %}
 
-{% include alert.html icon='info_outline'
-                      header='**Informational** *alert*'
-                      body='This is an **informational** alert *with* `<markdown/>`{:.language-html}.' %}
+{% include alert.html icon='info_outline' header='**Informational**     *alert*'
```
Comment (payex-dev) [inline index.md:186]: Codacy Issue found: [[emphasis-marker] Emphasis should use `_` as a marker](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108)

File: index.md (hunk 5 above)
Comment (payex-dev) [inline index.md:187]: Codacy Issue found: [[maximum-line-length] Line must be at most 80 characters](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 33
- Added 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108&bid=18291964)

File: index.md (hunk 8 above)
Comment (payex-dev) [inline index.md:186]: Codacy Issue found: [[emphasis-marker] Emphasis should use `_` as a marker](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108)

File: index.md (hunk 5 above)
Comment (payex-dev) [inline index.md:187]: Codacy Issue found: [[maximum-line-length] Line must be at most 80 characters](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108)

File: index.md [hunk 9]
```diff
@@ -208,37 +211,37 @@ and try out PayEx Checkout for yourself!' %}
...
-    * level 3 item
-* level 1 item
-  * level 2 item
-  * level 2 item
-  * level 2 item
-* level 1 item
-  * level 2 item
-  * level 2 item
-* level 1 item
+*   level 1 item
```
Comment (payex-dev) [inline index.md:232]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108)

File: index.md (hunk 5 above)
Comment (payex-dev) [inline index.md:187]: Codacy Issue found: [[emphasis-marker] Emphasis should use `_` as a marker](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108)

File: index.md [hunk 10]
```diff
@@ -208,37 +211,37 @@ and try out PayEx Checkout for yourself!' %}
...
+  *   level 2 item
```
Comment (payex-dev) [inline index.md:233]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108)

File: index.md [hunk 11]
```diff
@@ -208,37 +211,37 @@ and try out PayEx Checkout for yourself!' %}
...
+  *   level 2 item
+    *   level 3 item
+    *   level 3 item
+*   level 1 item
+  *   level 2 item
+  *   level 2 item
+  *   level 2 item
+*   level 1 item
+    *   level 2 item
```
Comment (payex-dev) [inline index.md:242]: Codacy Issue found: [[no-missing-blank-lines] Missing blank line before block node](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108)

File: index.md [hunk 12]
```diff
@@ -208,37 +211,37 @@ and try out PayEx Checkout for yourself!' %}
...
-  * level 2 item
-  * level 2 item
-* level 1 item
-  * level 2 item
-  * level 2 item
-* level 1 item
+*   level 1 item
+  *   level 2 item
+  *   level 2 item
+    *   level 3 item
```
Comment (payex-dev) [inline index.md:235]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5580108)

**Comment by NullableInt** [comment]
Let's merge it to master to see if Codacy will detect the settings file.

**Pull request merged**